Motion Amendments - January 2020 (#139)
thewca/wca-documents: documents/motions/08.2020.2 - Trainee Delegates.md

# SUBMISSION OF PROPOSED MOTION

**Motion number:** 8.2020.2  
**Subject:** WCA Trainee Delegates  
**Intent:** Procedure of acknowledgement of WCA Trainee Delegates and description of their rights and duties  
**Submitted by:** Board of Directors  
**Date:** January 31, 2020  

# Motion

Trainee Delegates are Community Members who have been appointed by the WCA to train for the role of WCA Delegate.

> 1. Trainee Delegates will require a time period of extra mentoring and assistance by Senior Delegates and/or Regional Delegates, to support their development to Candidate Delegates and Staff of the WCA.

Reviewer: Should this extra mentoring also be reflected in the Senior Delegates motion (e.g. in 3.2.3) and Regional Delegates motion? I would also like to see some of this mentoring be codified as the responsibility of all non-Trainee Delegates, since that is the purpose of requiring another Delegate. A Senior/Regional Delegate won't always be at the competition with the Trainee, so some of the responsibility of training them should fall to the Candidate/Full Delegate present.

Author: Clarified this in the upcoming amendments.

> 1. Trainee Delegates will require a time period of extra mentoring and assistance by Senior Delegates and/or Regional Delegates, to support their development to Candidate Delegates and Staff of the WCA.

Author: ```suggestion
1. Trainee Delegates will require a time period of extra mentoring and assistance by Senior Delegates and/or Regional Delegates, to support their development to Junior Delegates and Staff of the WCA.
```

>    1. A Senior Delegate may nominate a Trainee Delegate for a promotion to Candidate Delegate, if the Senior Delegate judges that the Trainee Delegate has shown their fitness for the position.

Author: ```suggestion
   1. A Senior Delegate may nominate a Trainee Delegate for a promotion to Junior Delegate, if the Senior Delegate judges that the Trainee Delegate has shown their fitness for the position.
```

>    2. To promote a Trainee Delegate to Candidate Delegate, the Senior Delegate must submit the corresponding application to the WCA Board.

Author: ```suggestion
   2. To promote a Trainee Delegate to Junior Delegate, the Senior Delegate must submit the corresponding application to the WCA Board.
```

   3. The WCA Board rejects or approves the application.

>    4. After approval, the Senior Delegate appoints the person as Candidate Delegate.

Author: ```suggestion
   4. After approval, the Senior Delegate appoints the person as Junior Delegate.
```

2. Rights of Trainee Delegates
   1. Trainee Delegates have the right to be listed as a Delegate at WCA Competitions in which there is at least one other WCA Delegate listed.

>       1. At least one listed Delegate must possess the position of Candidate Delegate, Full Delegate, Regional Delegate, or Senior Delegate.

Author: ```suggestion
      1. At least one listed Delegate must possess the position of Junior Delegate, Full Delegate, Regional Delegate, or Senior Delegate.
```

   2. Trainee Delegates have the right to make decisions at competitions in which they are listed as a Delegate.
	 3. Trainee Delegates have the right to read and participate in conversations in the Delegates and Reports groups.

> 	 4. Trainee Delegates have the right to contribute to and submit Delegate Reports about competitions that they helped oversee.

Reviewer 2: In my opinion, a trainee delegate should have the right to contribute to competition reports, but **not** to submit them. This ensures an overseeing delegate in the loops which enforces their training.

Author: Agreed. I suggested accordingly.

> 	 4. Trainee Delegates have the right to contribute to and submit Delegate Reports about competitions that they helped oversee.

Author: ```suggestion
	 4. Trainee Delegates have the right to contribute to Delegate Reports about competitions that they helped oversee.
```

> 	 5. Trainee Delegates have the right to submit results for competitions they helped oversee.

Reviewer 2: Similar to above, a trainee delegate should have the right to compile and upload JSON files, but **not** to submit them. This ensures an overseeing delegate in the loops which enforces their training.

Author: I made a suggestion. Does this work?

> 	 5. Trainee Delegates have the right to submit results for competitions they helped oversee.

Author: ```suggestion
	 5. Trainee Delegates have the right to review and compile results for competitions they helped oversee.
```

	 6. Trainee Delegates have the right to submit corrections and changes to results of competitions they helped oversee.
3. Duties of Trainee Delegates
   1. Trainee Delegates shall behave as a representative of the WCA in accordance with the Mission, Spirit, Regulations, Policies, and Code of Ethics of the WCA.
	 2. Trainee Delegates shall sustain knowledge and skills regarding the Mission, Spirit, and Regulations of the WCA.
	 3. Trainee Delegates shall accept and comply with the decisions of the WCA Board, WCA Officers, and WCA Teams/Committees.
	 4. For every WCA Competition that they are overseeing, Trainee Delegates:
	    1. represent the WCA

> 			2. make sure Competition Organizers and Competition Staff are fully capable, equipped, and prepared to make the competition successful

Reviewer 2: Formatting is broken here (overindented).

Author: Fixed in upcoming amendments.

> 			2. make sure Competition Organizers and Competition Staff are fully capable, equipped, and prepared to make the competition successful

Author: ```suggestion
		2. make sure Competition Organizers and Competition Staff are fully capable, equipped, and prepared to make the competition successful
```

> 			3. make sure the competition is run according to the Mission, Spirit, and Regulations of the WCA

Author: ```suggestion
		3. make sure the competition is run according to the Mission, Spirit, and Regulations of the WCA
```

> 			4. timely submit a report about the competition to the community of WCA Delegates

Author: ```suggestion
		4. timely submit a report about the competition to the community of WCA Delegates
```

> 			5. timely send in the results for the competition to the WCA Results Team

Author: ```suggestion
		5. timely send in the results for the competition to the WCA Results Team
```

> 			6. inform the WCA Results Team about any late corrections and changes in the results

Author: ```suggestion
		6. inform the WCA Results Team about any late corrections and changes in the results
```

> 			7. ensure timely payment of WCA Dues in cooperation with the other Delegates and Organizers

Reviewer: In line with the Dues policy 2.1.1 I would prefer for this to be in cooperation with other Delegates only (i.e. remove "and Organizers"). If the intention is to make the Trainee Delegate not solely responsible for Dues payment then that is very understandable, but Delegates are responsible for paying dues, with or without the cooperation of organisers.

As a side note: I notice the payment of WCA Dues is not a duty of Delegates per the Delegates motion, yet it has been included here. This is fine since the Dues policy outlines that it is the responsibility of the WCA Delegate to pay dues (and the Dues policy is a decision of the WCA Board), however it could be a suggestion to add a similar line to the Delegates motion?

Author: Yes, I think it makes sense to update in the Delegates Motion.

> 			7. ensure timely payment of WCA Dues in cooperation with the other Delegates and Organizers

Author: ```suggestion
		7. ensure timely payment of WCA Dues in cooperation with the other Delegates and Organizers
```

> 			7. ensure timely payment of WCA Dues in cooperation with the other Delegates and Organizers

Author: ```suggestion
			7. ensure timely payment of WCA Dues in cooperation with the other Delegates
```

4. Appointment of Trainee Delegates
   1. Senior Delegates may appoint suitable persons who meet the following requirements:
	    1. Full trustworthiness

> 			2. A high level of knowledge of the Mission, Spirit, and Regulations of the WCA

Reviewer 2: again, formatting is broken (overindented).

> 			2. A high level of knowledge of the Mission, Spirit, and Regulations of the WCA

Author: ```suggestion
		2. A high level of knowledge of the Mission, Spirit, and Regulations of the WCA
```

> 			3. Strong, independent, communicative, positive personality

Author: ```suggestion
		3. Strong, independent, communicative, positive personality
```

> 			4. Leadership with strong feeling of responsibility

Author: ```suggestion
		4. Leadership with strong feeling of responsibility
```

> 			5. Organizational skills, ability to perform core organizational tasks if necessary

Author: ```suggestion
		5. Organizational skills, ability to perform core organizational tasks if necessary
```

> 			6. Representativeness for the WCA

Author: ```suggestion
			6. Representativeness for the WCA
```
```suggestion
		6. Representativeness for the WCA
```
